### ISSUE TITLE
Some layout difference when in beta

### ISSUE THREAD

**Issue by zond** [zond/dipact#173]
We should have some layout difference in beta-dot-dipact, so nobody gets confused about whether they are in the beta.

@JorenC do you have ideas?

**Comment by JorenC** [comment]
We have the loading app screen (can I customize that?).. And we can use diplicity.com and beta.diplicity.com.

**Comment by zond** [comment]
I don't think we can use beta.diplicity.com - it's OK, I think, to let web beta users use beta-dot-dipact.appspot.com.

You can edit the splash screen if you checkout https://github.com/zond/android-dipact and change stuff for the app flavor "beta". AFAIU you can just make the values in the res folder different for prod and beta, and the two variants built will use the stuff in the correct res folder.

**Comment by JorenC** [comment]
It's already working, just with a 301 redirect. It's still easier to
remember and guide people to try beta after you updated a feature

**Comment by zond** [comment]
Sure :)
